### DISCUSSION TITLE
SNES-TXB0108

### DISCUSSION THREAD

**Discussion by ghost** [darthcloud/BlueRetro#173] [Q&A] [answered]
I'm trying to make the BlueRetro connection on a SNES, I'm trying it through a logic converter based on a TXB0108 chip, but at the moment it works badly for me (It only responds to pressing 2 X and Square buttons, I'm using a PS4 controller) and when doing so, the response I get is not always the same (that is, maybe pressing X on the control, for example, what I get as a response can be Start, like the up button, down button or any of the rest), you know if the TXB0108 is not can it be used? Or is it that I may be doing something wrong?

**Comment by darthcloud** [comment, marked as answer]
NEVER use bi-direction level shifter. Those chip are garbage and simply doesn't work except in some very special case I2C.

Get yourself some 74AHCT125 !!!

**Reply by ghost** [reply]
Thank you very much for the prompt response friend, I already ask for the suggested ones and as soon as they arrive I will tell you how it went, once again a thousand thanks
